.ci/build_kubernetes.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh -ex
```

Comment:
We should change name of this sh file, it does not make sense.

Reply:
I see it in docs where it was put into build section with info `Build of the application is then executed by one single command`.